systemtest/src/main/java/io/strimzi/systemtest/performance/report/parser/PerformanceMetricsParser.java

```diff
@@ -0,0 +1,392 @@
+                        final List<ExperimentMetrics> experimentsList = new ArrayList<>();
+                        final File[] experimentsFiles = useCaseDir.listFiles();
+
+                        if (experimentsFiles != null) {
+                            for (File experimentFile : experimentsFiles) {
+                                final File[] metricFiles = experimentFile.listFiles();
+                                final ExperimentMetrics experimentMetrics = new ExperimentMetrics();
+
+                                if (metricFiles != null) {
+                                    for (File metricFile : metricFiles) {
+                                        if (metricFile.getName().endsWith("test-performance-metrics.txt")) {
+                                            try {
+                                                processFile(metricFile, experimentMetrics);
+                                            } catch (IOException e) {
+                                                System.err.println("Error processing file " + metricFile.getAbsolutePath());
+                                                e.printStackTrace();
+                                            }
+                                            // these are most metrics
+                                        } else {
+                                            final List<Double> values = new ArrayList<>();
+                                            final List<String> lines = Files.readAllLines(metricFile.toPath());
+
+                                            for (String line : lines) {
+                                                final String[] parts = line.split(", Values: ");
+                                                if (parts.length > 1 && !parts[1].equals("[]")) {
+                                                    final String valuesPart = parts[1].replaceAll("\\[|\\]", ""); // Remove brackets
+                                                    final String[] valueStrings = valuesPart.split(", ");
+                                                    for (String valueString : valueStrings) {
+                                                        try {
+                                                            double value = Double.parseDouble(valueString);
+                                                            values.add(value);
+                                                        } catch (NumberFormatException e) {
+                                                            System.err.println("Error parsing value: " + valueString);
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                            // TODO: we can do post processing here.. (AVG, Q90, Q99 for strimzi_total_time_spend_on_uto_event_queue_duration_seconds f.e.)
```

Comment:
And will you do it?

Reply:
I think not in this PR I assume, I will create probably issue for this :)